.github/workflows/validation_tests.yml

```diff
@@ -0,0 +1,105 @@
+name: Connector Ops CI - Run Live Tests
+
+concurrency:
+  # This is the name of the concurrency group. It is used to prevent concurrent runs of the same workflow.
+  #
+  # - github.head_ref is only defined on PR runs, it makes sure that the concurrency group is unique for pull requests
+  #  ensuring that only one run per pull request is active at a time.
+  #
+  # - github.run_id is defined on all runs, it makes sure that the concurrency group is unique for workflow dispatches.
+  # This allows us to run multiple workflow dispatches in parallel.
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+    inputs:
+      connector_name:
+        description: Connector name (e.g. source-faker)
+        required: true
+      connection_id:
```

Comment:
Should connection_id be optional if we're supporting live validation tests too?

Reply:
We could set `auto` as a default. I think @clnoll original intent was to make users very intentional about which connection is used for testing.